### PR TITLE
ETC Course: 4. Satoshi Nakamoto and Bitcoin

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is class 4 of the ETC course where the origin of Bitcoin is explained and is related to ETC.

Chinese version will be coming soon.

This post will be published on 2003-01-12.

EDIT: Chinese version is ready.

### CODE REVIEW

**Review by w1g0 — APPROVED** [review]

File: content/blog/2023-01-12-ethereum-classic-course-4-satoshi-nakamoto-and-bitcoin/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+And, all these combined assure the immutability and security of the blockchain.
+
+## What Does it Accomplish?
+
+As said before, Nakamoto Consensus accomplishes a level of safety and security of the system never reached before while keeping it decentralized.
+
+This makes it trust minimized, meaning that users may interact directly between themselves in a peer-to-peer way, sending and receiving money, without the need of trusted third parties, who may be captured by special interests, restricting freedom and human action.
+
+Just as the internet, Bitcoin is designed to survive a nuclear war, and, on top of that, it assures hard money for the internet.
```
Comment (TheCrowbill) [inline content/blog/2023-01-12-ethereum-classic-course-4-satoshi-nakamoto-and-bitcoin/index.md:99]: Line 99: Corrected run-on sentence
```suggestion
Just like the internet, Bitcoin is designed to survive a nuclear war. On top of that, it assures hard money for the internet.
```

File: content/blog/2023-01-12-ethereum-classic-course-4-satoshi-nakamoto-and-bitcoin/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+## Implications for Ethereum Classic
+
+Ethereum Classic uses the same consensus mechanism as Bitcoin. However, ETC adds to the base technology of Bitcoin the ability to store and execute smart contracts or decentralized programs. This turns applications in ETC into decentralized applications which execute their code inside the highly secure environment of the blockchain. 
+
+So, ETC has the same two base pillars as Bitcoin; a fixed supply and the proof of work based consensus mechanism; but to that it added smart contracts. This makes ETC’s principle of Code Is Law a true statement. Therefore, ETC gives money and applications the benefits of soundness, survivability, censorship resistance, unstoppability, and they are uncompromisable and uncapturable.
```
Comment (TheCrowbill) [inline content/blog/2023-01-12-ethereum-classic-course-4-satoshi-nakamoto-and-bitcoin/index.md:105]: Line 105: Corrected run-on sentence
```suggestion
So, ETC has the same two base pillars as Bitcoin; a fixed supply and the proof of work based consensus mechanism; but to that it added smart contracts. This makes ETC’s principle of Code Is Law a true statement. Therefore, ETC gives money and applications the benefits of soundness, survivability, censorship resistance, and unstoppability making them uncompromisable and uncapturable.
```

**Review by TheCrowbill — DISMISSED** [review]
Good article!
Approved with suggested corrections.

**Review by bobsummerwill — APPROVED** [review]
Great article.  Thanks!

**Review by w1g0 — APPROVED** [review]

**Pull request merged**